crd/hack/verify-codegen.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly SCRIPT_ROOT=$(cd $(dirname ${BASH_SOURCE})/.. && pwd)
+echo "SCRIPT_ROOT ${SCRIPT_ROOT}"
+
+${SCRIPT_ROOT}/hack/update-codegen.sh
+
+# Test for diffs
+diffs=$(git status --porcelain | wc -l)
```

Comment:
Do we need this? I see that there is "VERIFY_CODEGEN" env which triggers some `--verify-only` mode.

Reply:
I really don't understand well the codegen scripts ... this is how is done in kubernetes and in other projects I´ve seen using this generators, if you run again and the git tree is the same means we are good

Reply:
Ok, that makes sense. I can't say that '--verify-only' will always work and git script makes more sense.